### ISSUE TITLE
[CI] RunDataFrameAnalyticsIT testLazyAssignmentWithModelMemoryLimitTooHighForAssignment failing on Debian 8

### ISSUE THREAD

**Issue by mark-vieira** [elastic/elasticsearch#68611]
Seen a number of these failures and they seem to exclusively happen on our Debian 8 CI agents... coincidence?

**Build scan**:
https://gradle-enterprise.elastic.co/s/vnfcm4cycvhik/tests/:x-pack:plugin:ml:qa:native-multi-node-tests:javaRestTest/org.elasticsearch.xpack.ml.integration.RunDataFrameAnalyticsIT/testLazyAssignmentWithModelMemoryLimitTooHighForAssignment#1

**Repro line**:
`./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:javaRestTest' --tests "org.elasticsearch.xpack.ml.integration.RunDataFrameAnalyticsIT.testLazyAssignmentWithModelMemoryLimitTooHighForAssignment" -Dtests.seed=64D580EABFA5F688 -Dtests.security.manager=true -Dtests.locale=es-HN -Dtests.timezone=America/Nuuk -Druntime.java=8`

**Reproduces locally?**:
Nope.

**Applicable branches**:
`7.x`, `7.11` and `7.10`

**Failure history**:
https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=America/Los_Angeles&tests.container=org.elasticsearch.xpack.ml.integration.RunDataFrameAnalyticsIT&tests.disabledDistributions=WyJvdXRjb21lOnBhc3NlZCJd&tests.sortField=FAILED&tests.test=testLazyAssignmentWithModelMemoryLimitTooHighForAssignment&tests.unstableOnly=true

**Failure excerpt**:
```
org.elasticsearch.xpack.ml.integration.RunDataFrameAnalyticsIT > testLazyAssignmentWithModelMemoryLimitTooHighForAssignment FAILED
    java.lang.AssertionError: 
    Expected: an empty string
         but: was "V6MWCf2VTZOZ5KaageG-0g"
        at __randomizedtesting.SeedInfo.seed([64D580EABFA5F688:5BB63344BA035AC5]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.xpack.ml.integration.RunDataFrameAnalyticsIT.testLazyAssignmentWithModelMemoryLimitTooHighForAssignment(RunDataFrameAnalyticsIT.java:562)
```

**Comment by droberts195** [comment]
It’s another side effect of failing to find out how much memory is on a Debian 8 node, like #66629 and #66685. Hopefully will be fixed on Monday by #68542.

**Comment by romseygeek** [comment]
Another failure, this time on 7.10:
https://gradle-enterprise.elastic.co/s/lklfvuue7qhcu/console-log?task=:x-pack:plugin:ml:qa:native-multi-node-tests:javaRestTest

#68542 has not been backported to 7.10.x yet

**Comment by benwtrent** [comment]
The last failure for this was ~3 days about (before the fix was backported to 7.11). 

The fix was not backported back to 7.10, but since there will no longer be release builds for 7.10 (now that 7.11 is released), closing this issue
